spec/data_migrations/change_state_for_passive_enrollment_spec.rb

```diff
@@ -14,8 +14,8 @@
     let(:organization) {
       org = FactoryGirl.create :organization, legal_name: "Corp 1"
       employer_profile = FactoryGirl.create :employer_profile, organization: org
-      renewing_plan_year = FactoryGirl.create :plan_year, employer_profile: employer_profile, aasm_state: :renewing_enrolled, :start_on => Date.new(calendar_year, 4, 1), :end_on => Date.new(calendar_year+1, 3, 31),
-      :open_enrollment_start_on => Date.new(calendar_year, 3, 1), :open_enrollment_end_on => Date.new(calendar_year, 3, 10), fte_count: 5
+      renewing_plan_year = FactoryGirl.create :plan_year, employer_profile: employer_profile, aasm_state: :renewing_enrolled, :start_on => Date.new(calendar_year, 3, 1), :end_on => Date.new(calendar_year+1, 2, 28),
```

Comment:
need to know if this rake task is valid moving forward, If the spec needs to be fixed we need to know if plan year factory is behaving improperly and why moving the dates fixes this issue. What model rule is this patching around?

Reply:
As per my understanding, this rake task says when employee have health and when he try to select dental, then the health is canceled, so the data was fixed and we won't get the same scenario again. We don't need this.

Reply:
still didnt' answer why we are moving the dates around in this one